dev/AppNotifications/NotificationProperties.cpp

```diff
@@ -47,6 +49,14 @@ NotificationProperties::NotificationProperties(winrt::AppNotification const& toa
     {
         m_toastProgressData = winrt::make_self<NotificationProgressData>(toastNotification.Progress());
     }
+
```

Comment:
Please add Velocity changes too in new code: https://github.com/microsoft/WindowsAppSDK/blob/6e225c914bdad3a63f63459b0413b053026a34fe/docs/Coding-Guidelines/Experimental.md

Reply:
@anupriya13 , Added velocity containment changes in the recent commit , please verify.